app/src/Pages/Portal/Contributions/ContributionsTable/ContributionsTable.js

```diff
@@ -64,7 +64,7 @@ const columns = isGovAdmin => [
 const ContributionsTable = ({ ...props }) => {
   const isLoading =
     props.isListLoading && !Array.isArray(props.contributionList);
-  const title = `${props.total} Contributions`;
+  const title = `${props.total} Total Contributions`;
```

Comment:
Removed the word Total. It can represent both total or a subset.

Reply:
If total is not there it will be miss-leading though

Reply:
How about Total Contributions Results? There was a discussion about this and the thought was that in a limited set Total was confusing.